src/test/java/com/dev/bootbasic/voucher/repository/InMemoryVoucherRepositoryTest.java

```diff
@@ -0,0 +1,42 @@
+package com.dev.bootbasic.voucher.repository;
+
+import com.dev.bootbasic.voucher.domain.FixedAmountVoucher;
+import com.dev.bootbasic.voucher.domain.PercentDiscountVoucher;
+import com.dev.bootbasic.voucher.domain.Voucher;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class InMemoryVoucherRepositoryTest {
+
+    private final VoucherRepository voucherRepository = new InMemoryVoucherRepository();
+
+    @DisplayName("인메모리 바우처 저장소에 바우처를 저장할 수 있다.")
```

Comment:
```suggestion
    @DisplayName("바우처를 저장할 수 있다.")
```

정도면 충분한 것 같아요. 클래스 이름 자체가 인메모리 저장소인 것을 나타내고 있어서요

Reply:
반영했습니다. [ba33e97](https://github.com/prgrms-be-devcourse/springboot-basic/pull/676/commits/ba33e97e9a4245b4a710c68c773169c8b9e13000)